proto/tabletmanagerdata.proto

```diff
@@ -682,6 +682,8 @@ message UpdateVReplicationWorkflowRequest {
   binlogdata.OnDDLAction on_ddl = 5;
   binlogdata.VReplicationWorkflowState state = 6;
   reserved 7; // unused, was: repeated string shards
+  string stop_position = 8;
+  repeated int64 include_ids = 9;
```

Comment:
I'm failing to see where these two new fields are really being used except in tests. Can you point me to the actual usage?

Reply:
I was initially going to use it in this work but then did not end up using it. I left it in only because I know what we'll soon be leveraging this as we move more queries away from VReplicationExec there's a whole group of those which set the StopPosition for 1 or more streams/IDs.

Reply:
Removed here: https://github.com/vitessio/vitess/pull/15536/commits/20ed6beef8bb7a34dc68a8e3284170f4809198a8

It's a single commit so we can easily resurrect it later.